<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecutionContract</class>
 <widget class="QWidget" name="ExecutionContract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Название</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Дата начала</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="DateStart"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="Contract"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="FIO"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Оформить контракт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
